package.json

```
@@ -1,83 +1,71 @@
 {
   "name": "pci-dss-sanitizer",
-  "version": "0.0.0-development",
+  "version": "0.0.0-development.0",
   "description": "Mask bank card info from strings",
-  "main": "lib/index.js",
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint",
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "npm run check-coverage"
-    }
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "engines": {
+    "node": "16"
   },
+  "peerDependencies": {},
   "scripts": {
-    "prebuild": "npm run lint && rimraf lib",
-    "lint": "eslint --cache src/**/*.js",
-    "format": "npm run lint -- --fix",
-    "build": "rollup -c",
-    "watch": "rollup -c --watch",
-    "test": "cross-env NODE_ENV=test jest",
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "npm run test:client && npm run test:src --",
```

I tested if the thresholds would be checked on the CI, but this is currently not the case.

```suggestion
    "test": "npm run test:client && npm run test:src -- --coverage",
```